### PR TITLE
Updated installation steps for Linux

### PR DESCRIPTION
**Pull request by danroth27**

### CODE REVIEW

**Comment by danroth27** [comment]
@BrennanConroy @glennc Please review

File: aspnet/getting-started/installing-on-linux.rst [hunk 1]
```diff
@@ -24,24 +20,50 @@ The rest of this section deals with setting up a machine to run applications wit
...
+The following instructions were tested using Ubuntu 14.04.
+
+Install the .NET Version Manager (DNVM)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Use the .NET Version Manager (DNVM) to install different versions of the .NET Execution Environment (DNX) on Linux.
+
+To install DNVM run the following::
+
+    curl -sSL https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.sh | DNX_BRANCH=dev sh && source ~/.dnx/dnvm/dnvm.sh
```
Comment (BrennanConroy) [inline aspnet/getting-started/installing-on-linux.rst]: Should there be something about "permanently" sourcing dnvm, i.e. putting it in .bashrc or .bash_profile

Comment (danroth27) [inline aspnet/getting-started/installing-on-linux.rst]: The dnvminstall.sh script seems to take care of that

**Comment by BrennanConroy** [comment]
:shipit:

**Pull request closed without merging**